api/src/pcapi/core/bookings/models.py

```diff
@@ -153,13 +161,19 @@ class Booking(PcObject, Base, Model):
         "Deposit", foreign_keys=[depositId], back_populates="bookings"
     )
 
-    def mark_as_used(self) -> None:
+    def mark_as_used(self, author_set_used: BookingAuthorType) -> None:
```

Comment:
```suggestion
    def mark_as_used(self, validation_author_type: BookingAuthorType) -> None:
```
Tu remplis `validationAuthorType`, pourquoi ne pas mettre `validation_author_type` ?